subsys/bluetooth/host/gatt.c

```diff
@@ -1628,6 +1628,11 @@ void bt_gatt_cb_register(struct bt_gatt_cb *cb)
 	sys_slist_append(&callback_list, &cb->node);
 }
 
+void bt_gatt_cb_unregister(struct bt_gatt_cb *cb)
+{
+	sys_slist_find_and_remove(&callback_list, &cb->node);
```

Comment:
We are delightfully inconsistent w.r.t. return values for `cb_unregister` functions 🙃 

It does like we are slightly leaning towards to having them as `int` with NULL checks and also a non-0 return value if `cb` isn't registered in the first place.